akka-actor/src/main/scala/akka/actor/RepointableActorRef.scala

```diff
@@ -222,6 +226,8 @@ private[akka] class UnstartedCell(val systemImpl: ActorSystemImpl,
         if (cellIsReady(cell)) {
           cell.sendMessage(msg)
         } else if (!queue.offer(msg)) {
+          println(s"# enqueuedNonSystemMsg $msg") // FIXME
+          enqueuedNonSystemMsg = true
```

Comment:
Have we considered keeping a var to a SystemMessage that we prepend to until we repoint it, where we reverse it and send them (the SystemMessages) to the "real" actor ref?